### DISCUSSION TITLE
_GOV in facts and events

### DISCUSSION THREAD

**Discussion by webtrees-pesz** [vesta-webtrees-2-custom-modules/vesta_gov4webtrees#98] [Q&A] [answered]
How do I make the field for GOV ID visible for places in input forms if I have only GOV4webtrees enabled but not the module for SharedPlaces?

Best regards, Peter

**Comment by ric2016** [comment, marked as answer]
This use case isn't supported by the Gov4Webtrees module. 

A default edit control should be available when activating the GEDCOM-L tags in the control panel, but this only works for existing _GOV subtags (this part is a webtrees bug).

In any case enabling or disabling Shared Places does not affect how  3 _GOV under 2 PLAC is handled, nor should it.

Edit: See also [this issue](https://github.com/vesta-webtrees-2-custom-modules/vesta_gov4webtrees/issues/18). I'm undecided whether this should be supported. In general it doesn't seem like a good idea to add GOV Ids to single events without a more global management.

**Reply by webtrees-pesz** [reply]
> I'm undecided whether this should be supported. In general it doesn't seem like a good idea to add GOV Ids to single events without a more global management.

Yes, you are right and I usually see it that way too. But in this specific case the places of the family tree are still unstructured (incomplete, partly wrong hierarchy, sometimes only address) and need to be revised before shared places can be generated from it.

**Comment by webtrees-pesz** [comment]
Greg, solved it today. See https://github.com/fisharebest/webtrees/issues/4812

Best regards, Peter
